hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -608,6 +614,11 @@ private void loginAsSCMUser(ConfigurationSource conf)
   }
 
 
+  // TODO: Ratis is always enabled now. In case, ratis gets disabled, this
+  // switch should be adjusted accordingly.
+  public static boolean isRatisEnabled(OzoneConfiguration conf) {
```

Comment:
There is a utility method SCMHAUtils#isSCMHAEnabled we can use this and change the default value of OZONE_SCM_HA_ENABLE_DEFAULT to true, as now ratis is default looks like. (I see there is no usage for this method)

Reply:
Thanks @bharatviswa504 . The idea here is to check HA enablement but, whether ratis is being used even for non-HA mode. Its a placeholder code for now and the logic to invoke code with/without ratis for non-HA setup needs to be worked upon yet.

Reply:
If that is the use case isSCMHAEnabled we can use it right, as that is the config to let know whether ratis is used in SCM or not.

Reply:
Done